### PR TITLE
Add, remove and edit functionalities

### PR DESCRIPTION
**Pull request by HansZizold**
Add, remove and edit functionalities added to the project.
Index.html and Style.css adjusted to match the original design.
JS files modified to use modules and ES6 syntax.
Readme.md file updated.
Webpack used to pack the project.
Linter files included.

### CODE REVIEW

**Comment by HansZizold** [comment]
I have a strange error on linters on Github.  I don't know the reason, but when I run the linters locally I have no errors as you can see in the printout:
image

Please take this into consideration.

File: src/index.html [hunk 1]
```diff
@@ -4,24 +4,23 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css">
+  <!-- <link rel="stylesheet" href="./style.css"> -->
```
Comment (youmari) [inline src/index.html]: - [ ] please try to remove the unused commented code in order to keep the file clean and professional

File: src/index.js [hunk 2]
```diff
@@ -1,24 +1,67 @@
...
+      completed: false,
+      index: checkbox.length + 1,
+    };
+    taskArray.push(taskObject);
+    taskDescription.value = null;
+    // update local storage
+    localStorage.setItem('mytasks', JSON.stringify(taskArray));
+    checkbox = document.querySelectorAll('.checkbox');
+  }
+});
```
Comment (youmari) [inline src/index.js:30]: - [ ] when click on the checkbox and mark a task `completed: true` you should update the local storage data cause when I reload the page it becomes `completed: false`  please try to fix that

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @HansZizold , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Great title and short summary

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 2 above)
Comment (HansZizold) [reply src/index.js:30]: Hi  @youmari , thank you for the observations. 
You're right, I update the local storage but I am not updating the completed key. I will do that soon.

**Comment by HansZizold** [comment]
I included the update of the key 'completed' and the update of LS, so that it shows' true' when a task is marked. If the mark is removed it shows 'false' again. Also considered the case of reloading the page. I hope it is OK.
Thank you again for the correct observation.
Again linters on GITHUB fail, so I send you my local print screen:
image

**Review by smunozmo — APPROVED** [review]
Hi @HansZizold, 👋 

## Highlights 👏 

- Task status is updated in the localStorage, great!
- Add/Remove functionalities are working properly, good job!
- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

### APPROVED! 💯  💯  💯 




Cheers and Happy coding! 🤓 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**